packages/rest/jest.config.ts

```diff
@@ -6,7 +6,7 @@ const config: Config.InitialOptions = {
   ...base,
   name: '@aries-framework/rest',
   displayName: '@aries-framework/rest',
-  testTimeout: 120000,
+  testTimeout: 12000,
```

Comment:
My bad, these should have been changed back

```suggestion
  testTimeout: 120000,
```